src/roiextractors/extractors/schnitzerextractor/extractsegmentationextractor.py

```diff
@@ -169,11 +167,6 @@ def _trace_extractor_read(self) -> DatasetView:
         """Returns the traces with a shape of number of ROIs and number of frames."""
         return DatasetView(self._output_struct["temporal_weights"])
 
-    def _summary_image_read(self) -> np.ndarray:
-        """Returns the summary image as a numpy array where the first dimension is the
-        height of the frame and the second dimension is the width of the frame."""
-        return self._info_struct["summary_image"][:].transpose()
-
```

Comment:
This assumption was based off the previous EXTRACT segmentation extractor, but I'm not convinced that they are the same. There is not much information about parameter descriptions in the info struct, only that they contain algorithm statistics. Until it is confirmed I'd remove extracting it and reach out in the meantime to the developers whether what they call "summary image" is indeed the correlation image.

Reply:
[EXTRACT_output_to_nwb.m](https://github.com/schnitzer-lab/EXTRACT-public/blob/master/EXTRACT/helper_functions/EXTRACT_output_to_nwb.m#L208) 
Based on their export to NWB, they write: `summary_image`, `F_per_pixel`, and `max_image` to an Images container:
https://github.com/schnitzer-lab/EXTRACT-public/blob/d7a83736918fbef24643860ec6ac3aabfb71c617/EXTRACT/helper_functions/EXTRACT_output_to_nwb.m#L208-L210
To add these images I think we could add them as private attributes to the `NewSegmentationExtractor` class, and override the abstract class `SegmentationExtractor` `get_images_dict()` method, such that we could write these images to NWB. 

https://github.com/catalystneuro/roiextractors/blob/1f8a479e90a45595de661ad339fa53fc022294e2/src/roiextractors/segmentationextractor.py#L203-L212

What do you think about this @CodyCBakerPhD or @bendichter ?